ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/ContextExtensions.kt

```diff
@@ -14,3 +18,18 @@ internal fun Context.getActivity(): Activity? {
     }
     return null
 }
+
+private const val MAX_CACHE_SIZE_BYTES = 100 * 1024 * 1024L // 100 MB
```

Comment:
Maybe this is too much? Lmk what you think

Reply:
Is this a cache disk?
But yeah regardless probably too much. I'd say 25MB is probably enough.

Reply:
Yeah, disk cache. And yeah, I agree it was too much. Reduced to 25MB 👍